lib/reverse_sentence.rb

```diff
@@ -1,4 +1,60 @@
 # A method to reverse the words in a sentence, in place.
+require 'pry'
 def reverse_sentence(my_sentence)
-  raise NotImplementedError
+  # binding.pry
+  if my_sentence == nil || my_sentence.length == 1 || my_sentence.length == 0
+    return my_sentence
+  end
+  my_sentence = reverse_words(my_sentence)
```

Comment:
Since reverse_words is doing all the string manipulations in-place and so is string_reverse, you don't need to return anything from those methods or assign them in this method.

Lines 8 and 9 can simply be:
```ruby
   reverse_words(my_sentence)
   string_reverse(my_sentence, 0, (my_sentence.length - 1))
```